src/Examples/CIFAR10.cs

```diff
@@ -103,8 +105,8 @@ internal static void Main(string[] args)
             using (Dataset train_data = torchvision.datasets.CIFAR100(datasetPath, true, download: true),
                            test_data = torchvision.datasets.CIFAR100(datasetPath, false, download: true))
             {
-                using var train = new DataLoader(train_data, _trainBatchSize, device: device, shuffle: true);
-                using var test = new DataLoader(test_data, _testBatchSize, device: device, shuffle: false);
+                using var train = DataLoader(train_data, _trainBatchSize, device: device, shuffle: true);
```

Comment:
Trying to follow along with what's going on here, as this doesn't follow typical .NET conventions...

Is DataLoader here a static method defined on one of the types that's already being imported with a using static (I guess TorchSharp.torch.utils.data)? And then the `using TorchSharp.Modules;` is because somewhere in this file `DataLoader` is being used for its type name rather than for this static method name?